src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -336,20 +332,16 @@
     <Compile Include="System\IO\BinaryReader.cs" />
     <Compile Include="System\IO\DirectoryNotFoundException.cs" />
     <Compile Include="System\IO\EndOfStreamException.cs" />
-    <Compile Include="System\IO\Error.cs" />
     <Compile Include="System\IO\FileAccess.cs" />
     <Compile Include="System\IO\FileLoadException.cs" />
     <Compile Include="System\IO\FileMode.cs" />
     <Compile Include="System\IO\FileNotFoundException.cs" />
     <Compile Include="System\IO\FileShare.cs" />
     <Compile Include="System\IO\FileOptions.cs" Condition="'$(IsProjectNLibrary)' != 'true'" />
-    <Compile Include="System\IO\FileStream.cs" Condition="'$(IsProjectNLibrary)' != 'true'" />
```

Comment:
Could you please keep these `IsProjectNLibrary` in the shared file for now? The ProjectN TFS side would break without them currently.